courses/free-courses-en.md

```diff
@@ -381,6 +381,7 @@
 * [A Practical Introduction to Cloud Computing](https://www.udemy.com/course/introduction-cloud-computing/) - Neil Anderson
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Cloud Computing intro and applications](https://www.youtube.com/watch?v=G9LSFoBA-Bo&list=PLROvODCYkEM8xUk8R2T79KbZ7YfCXMbZS&index=1) - OU Education
```

Comment:
We prefer 'playlist' links for YouTube contributions, and refrain from creating new titles. Use the title provided by the source.
```suggestion
* [Cloud Computing (CC)](https://www.youtube.com/playlist?list=PLROvODCYkEM8xUk8R2T79KbZ7YfCXMbZS) - OU Education
```